releases/mainnet/1.9.2/index.html

```diff
@@ -0,0 +1,25 @@
+<html>
+	<head>
+		<title>Mainnet Release 1.9.2</title>
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Mainnet Release 1.9.2</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.9.2/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmXXTKKP8odp768F4EShHaFGsMmv55Dz1gsxTesKkfYUgU">Contract addresses (IPFS hosted)</a></li>
```

Comment:
Note to self: verify that IPFS link has changed once pips are available.

Reply:
IPFS link doesn't match mainnet contract addresses

Reply:
updated now and mirroring it around.

Reply:
Contents of IPFS link match the addresses in this PR (including pips).